pkg/cmd/auth/gitcredential/helper.go

```diff
@@ -55,6 +55,7 @@ func NewCmdCredential(f *cmdutil.Factory, runF func(*CredentialOptions) error) *
 	return cmd
 }
 
+// TODO: In multi-account we should use active user token only if the username is not passed in.
```

Comment:
This is not a `TODO`, this is what we agreed upon as behaviour for now. Will fix up the doc.